spec/cell_spec.rb

```diff
@@ -55,4 +58,44 @@
       expect(@cell.fired_upon?).to eq(true)
     end
   end
+
+  describe "#render" do
+    it "”.” if the cell has not been fired upon" do
+      expect(@cell_1.render).to eq(".")
+      expect(@cell_2.render).to eq(".")
+    end
+
+    it "“M” if the cell has been fired upon and it does not contain a ship" do
```

Comment:
Lets consider adding "can render" or "renders" to the beginning of these it statements.